pkg/pillar/scripts/veth.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -e
+
+try() {
+   if ! "$@"; then
+      ip link del "$VIF_CTR" || :
+      ip link del "$VIF_NAME" || :
+      exit 1
+   fi
+}
+
+VIF_NS=$(jq '.pid')
+case $1 in
+   down) ip link del "$2"
+         ;;
```

Comment:
Why don't you exit here as oppsed to trying the ip link add etc?

Reply:
`<embarrassment>man, that's a silly mistake</embarrassment>`

thanks for catching it @eriknordmark -- fixed!